expected-failures/stack.txt

```diff
@@ -0,0 +1,18 @@
+src/Stack/Path.hs
```

Comment:
Have you looked at this? Why does it happen?

Reply:
See the description of this PR. I am not sure whether to fix this in this PR or create a separate issue.